app/templates/components/modals/paytm-otp.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="header">
+  {{t 'Amount to be paid:'}} {{currency-symbol currency}} {{amount}}
+</div>
+
+<div class="content">
+  {{t 'Enter OTP sent to mobile number'}}
+  <div class="ui {{if isLoading 'loading'}} form" autocomplete="off">
+    {{input type='number' name='otp' value=otp required=true}}
+  </div>
+</div>
+
+<div class="actions">
+  <button class="ui green button" disabled={{isOTPvalid}}>
+    {{t 'Verify'}}
+  </button>
```

Comment:
@uds5501 I think that there should be a cancel button for this modal too

Reply:
@shreyanshdwivedi I have updated the modal according to the suggestion, please review now